io/disk/port_bounce.py

```diff
@@ -128,7 +128,7 @@ def run_command(self, command, timeout=300):
             self.fail("telnet connection to the fc/nic switch not yet done")
         self.remote_conn.send(command + '\n')
         time.sleep(self.verify_sleep_time)
-        response = self.remote_conn.recv(4000)
+        response = self.remote_conn.recv(8192)
```

Comment:
@maramsmurthy can you please paste the what exact error we see with existing value?
it will help for reference.

any reason for exact 8192 or we can make it larger than this?
what will be effect if we make larger value than 8192?

Reply:
@Naresh-ibm As script didn't find the expected string from recv text, we are are getting following error messages
'2': 'Port 2 is failed to Disabled'
'1': 'Port 1 is failed to Disabled

The value represents number of bytes. I just matched it to 8KB.

Reply:
We faced this problem during RHEL 8.8 and RHEL 9.2 configuration time also with newly configured 64Gig switch with 64 ports. Same scenario worked manually when I tried. 
This time debugged further and identified the problem. Sent this resolution as part of it.

Reply:
If the number port increases then find the expected string from recv text will also need to be increased